README.md

```diff
@@ -1,2 +1,54 @@
-# Hello-world
-testing
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# HELLO MICROVERSE
+
+> setup to test git features .
+
+![image](https://user-images.githubusercontent.com/59481426/148063440-51643a84-4032-490a-b93c-0a90381c8b17.png)
+
+
+
+
+## Built With
+
+- HTML,CSS
+
+- HTML,CSS,GITHUB
+
+## Live Demo
+
+[Live Demo Link](https://lucidkode.github.io/Hello-world/)
+
+
+
+
+
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Lucidkode)
+
```

Comment:
## [OPTIONAL]
- [ ] In the author section I think it would better add more contacts so that the visitor will easily get in touch with you. follow the example below 
```
- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```